extensions/adobe/experience/decisioning/decisoinitem.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/decisionitem",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Item",
+  "description": "An item which can be subjected to decisioning. The output of a decisioning process is one or more decision items.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/experience/decisioning/offeritem", "https://ns.adobe.com/xdm/classes/product"],
+  "definitions": {
+    "decisionitem": {
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemName": {
+          "title": "Decision Item Name",
+          "type": "string",
+          "description": "Name of a decision item"
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemDescription": {
+          "title": "Decision Item Description",
+          "type": "string",
+          "description": "Description of a decision item"
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemLabels": {
+          "title": "Decision Item Labels",
+          "type": "array",
+          "description": "OLAC labels for decision item.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "https://ns.adobe.com/experience/decisioning/decisionitem/itemIdentityMap": {
+          "$ref": "https://ns.adobe.com/xdm/context/identitymap",
```

Comment:
"identityMap" is a field group so, it can't be used inside a custom property as a reference. Please remove this property. I see that you have already included "identityMap" in the allOf in the end of this field group.